kong/plugins/acme/schema.lua

```diff
@@ -271,6 +271,14 @@ local schema = {
         then_err = "terms of service must be accepted, see https://letsencrypt.org/repository/",
       }
     },
+    { conditional = {
```

Comment:
I moved the validation from standardized redis config to where it's used. See this comment: https://github.com/Kong/kong/pull/12447/files#r1469635044